contracts/crowdsale/price/CapStagedCrowdsale.sol

```diff
@@ -0,0 +1,70 @@
+pragma solidity ^ 0.4.24;
+
+import "../Crowdsale.sol";
+import "../../math/SafeMath.sol";
+import "../../ownership/Ownable.sol";
+
+/**
+ * @title CapStagedCrowdsale
+ * @dev Extension of Crowdsale contract that changes the price of tokens regarding of raised ETH. Each stage of crowdsale is defined in StageLimit (cap limit for the stage) and stageRate (stage rate)
+ * Stages must be inserted from first to last with increasing stageLimit.
+ */
+contract CapStagedCrowdsale is Crowdsale, Ownable {
+  using SafeMath for uint256;
+  using SafeMath for uint;
+
+  struct Stage {
+    uint256 stageLimit;
+    uint256 stageRate;
+  }
+
+
+
+  Stage[] public stages;
+
+  constructor
+  (
+    uint256 _rate,
+    address _wallet,
+    ERC20 _token
+  )
+  Crowdsale(_rate, _wallet, _token) 
+  public
+  {
+  }
+
+  /**
+  * @dev Function for adding stages. Stages must be inserted from first to last
```

Comment:
nit: all of the comment formatting should be 

```
/**
 * ..
 * ..
 */
```

Reply:
Corrected and committed